docs/run_in_existing_vpc.md

```diff
@@ -1,8 +1,8 @@
 ## Running in a shared VPC
 
-When launching into a shared VPC, the VPC & the Internet Gateway will be reused. If you are not using an internet gateway
- or NAT gateway you can tell _kops_ to ignore egress. By default we create a new subnet per zone, and a new route table, 
- but you can also use a shared subnet (see [below](#shared-subnets)).
+When launching into a shared VPC, _kops_ will reuse the VPC and Internet Gateway. If you are not using an Internet Gateway
```

Comment:
Nit: do we need to emphasise kops?
If so, we should make that a convention for all our docs.

Reply:
I was trying to be consistent within the doc, but better to be consistent with the other docs. I took the italics out.